dace/transformation/transformation.py

```diff
@@ -243,12 +324,12 @@ def apply_to(cls,
                                  'given subgraph ("can_be_applied" failed)')
 
         # Apply to SDFG
-        instance.apply_pattern(sdfg)
+        return instance.apply_pattern(sdfg)
 
-    def __str__(self):
+    def __str__(self) -> str:
         return type(self).__name__
 
-    def print_match(self, sdfg):
+    def print_match(self, sdfg: SDFG) -> str:
```

Comment:
SDFG without module name

Reply:
I won't be able to change everything in this PR, especially not SDFG and SDFGState which exist pretty much everywhere.